pkg/operator/sync.go

```diff
@@ -528,6 +538,55 @@ func (optr *Operator) applyManifests(config *renderConfig, paths manifestPaths)
 	return nil
 }
 
+// safetySyncControllerConfig is a special case render of the controllerconfig that we run when
+// we need to keep syncing controllerconfig, but something (like daemon sync failing) prevents
+// us from getting to the actual controller sync
+func (optr *Operator) safetySyncControllerConfig(config *renderConfig) error {
+	glog.Infof("Performing safety controllerconfig sync")
+
+	// If we have an existing controllerconfig, we might be able to keep rendering
+	existingCc, err := optr.ccLister.Get("machine-config-controller")
```

Comment:
Minor nit: we have a const `ControllerConfigName` for this I believe

Reply:
we do indeed, thanks for catching it